src/libs/ajax.js

```diff
@@ -419,23 +413,7 @@ export const Election = {
       return {};
     }
     return await res.json();
-  },
-
-  updateElection: async (electionId, document) => {
-    const url = `${await getApiUrl()}/api/election/${electionId}`;
-    const res = await fetchOk(url, fp.mergeAll([Config.authOpts(), Config.jsonBody(document), { method: 'PUT' }]));
-    return await res.json();
-  },
-};
-
-export const Email = {
-
-  sendReminderEmail: async (voteId) => {
-    const url = `${await getApiUrl()}/api/emailNotifier/reminderMessage/${voteId}`;
-    const res = await fetchOk(url, fp.mergeAll([Config.authOpts(), { method: 'POST' }]));
-    return res;
   }
-
 };
```

Comment:
We have a ticket in the sprint to use this function ... might not want to delete it quite yet.